### DISCUSSION TITLE
StarCoder LLM is out! 100% coding specialized

### DISCUSSION THREAD

**Discussion by sarutobiumon** [oobabooga/text-generation-webui#1875] [General]
Really hope to see more specialized models becoming more common than general use ones, like one that is a math expert, history expert...etc

Hope it can run on WebUI, please give it a try!

mayank31398/starcoder-GPTQ-4bit-128g
https://huggingface.co/models?search=starcoder

**Comment by silvestron** [comment]
How do you run it?
